website/project/views/node.py

```diff
@@ -724,6 +726,23 @@ def _view_project(node, auth, primary=False,
     NodeRelation = apps.get_model('osf.NodeRelation')
 
     is_registration = node.is_registration
+
+    rejected_submissions = CollectionSubmission.objects.filter(
```

Comment:
Does it make sense to filter the `node.collecting_metadata_qs` QuerySet instead of creating a new QuerySet here? There probably won't be any performance gains but will probably make things more readable?

Reply:
Good suggestion, but we are trying to get away from the "collected metadata" verbiage for collection submissions